README.org

```diff
@@ -115,14 +150,33 @@ For example:
 
 * Using it with Chemacs2
 
-If you have the Chemacs2 configuration cloned to =~/.emacs.d= or =~/.config/emacs=, you can clone =rational-emacs= anywhere you like and add an entry to it in your =~/.emacs-profiles.el= file:
+If you have the Chemacs2 configuration cloned to =~/.emacs.d= or
+=~/.config/emacs=, you can clone =rational-emacs= anywhere you like
+and add an entry to it in your =~/.emacs-profiles.el= file:
+
+You can then put your =early-config.el= and =config.el= files in the
+subfolder =~/path/to/rational-emacs/rational-emacs=. So, for example
```

Comment:
Maybe for the case where they put the config under the repo folder, we can recommend the subfolder name `config`, like `~/path/to/rational-emacs/config`, or possibly `personal`?